src/client/datascience/jupyter/kernels/helpers.ts

```diff
@@ -784,7 +783,7 @@ export async function sendTelemetryForPythonKernelExecutable(
         });
         const execOutput = await execService.exec(['-c', 'import sys;print(sys.executable)'], { throwOnStdErr: false });
         if (execOutput.stdout.trim().length > 0) {
-            const match = execOutput.stdout.trim().toLowerCase() === sysExecutable;
+            const match = areInterpreterPathsSame(execOutput.stdout.trim().toLowerCase(), sysExecutable);
```

Comment:
Better way to check if interpreters are the same, we already use this same function a few lines above (line 764)